controllers/SelectSubmissionsListHandler.inc.php

```diff
@@ -0,0 +1,38 @@
+<?php
+/**
+ * @file classes/controllers/list/submissions/SelectSubmissionsListHandler.inc.php
+ *
+ * Copyright (c) 2014-2019 Simon Fraser University
+ * Copyright (c) 2000-2019 John Willinsky
+ * Distributed under the GNU GPL v2. For full terms see the file docs/COPYING.
+ *
+ * @class SelectSubmissionsListHandler
+ * @ingroup classes_controllers_list
+ *
+ * @brief A list handler for selecting submissions
+ */
+import('controllers.list.submissions.SubmissionsListHandler');
+
+class SelectSubmissionsListHandler extends SubmissionsListHandler {
```

Comment:
@NateWr, do you have a recommendation for an existing component for this? If not, would this be better added to the ui-library repo?

Reply:
There's been a change between stable and `master`. The list panels now take `canSelect` and `canSelectAll` arguments directly, and the components are no longer called "handlers".

```php
$listPanel = new \APP\components\listPanels\SubmissionsListPanel($id, $title, [
  'canSelect' => true,
  'canSelectAll' => true,
  ...
]);
```

Reply:
This file was actually added by accident. It is not being used, I'll remove it.